Tests/ClientTests/Ecosia/VersionTests.swift

```diff
@@ -0,0 +1,91 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+@testable import Client
+
+final class VersionTests: XCTestCase {
+    
+    override class func setUp() {
+        UserDefaults.standard.removeObject(forKey: Version.appVersionUpdateKey)
+    }
+    
+    override class func tearDown() {
+        UserDefaults.standard.removeObject(forKey: Version.appVersionUpdateKey)
```

Comment:
I guess there is no reason for `setUp` and `tearDown` to be the same? Can't you remove one of them?

Reply:
Sure will do! Thanks for that 👀